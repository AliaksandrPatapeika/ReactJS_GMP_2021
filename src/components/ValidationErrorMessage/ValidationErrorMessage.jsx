import './ValidationErrorMessage.less';

import PropTypes from 'prop-types';
import React from 'react';

function ValidationErrorMessage({children}) {
  return <span className="validationErrorMessage">{children}</span>;
}

ValidationErrorMessage.propTypes = {
  children: PropTypes.node.isRequired
};

export default ValidationErrorMessage;
