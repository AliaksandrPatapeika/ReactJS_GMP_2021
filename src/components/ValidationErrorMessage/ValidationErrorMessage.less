@import "../../less/variables";

.validationErrorMessage {
	color: @required-label-color;
	font-size: 15px;
	font-family: @NetflixSans-Regular;
}