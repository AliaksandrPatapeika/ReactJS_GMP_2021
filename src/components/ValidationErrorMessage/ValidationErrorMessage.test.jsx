import React from 'react';
import renderer from 'react-test-renderer';

import ValidationErrorMessage from './ValidationErrorMessage';

describe('ValidationErrorMessage', () => {
  test('renders correctly', () => {
    const validationErrorMessage = renderer
      .create(
        <ValidationErrorMessage>
          Validation error message
        </ValidationErrorMessage>
      )
      .toJSON();

    expect(validationErrorMessage).toMatchSnapshot();
  });
});
