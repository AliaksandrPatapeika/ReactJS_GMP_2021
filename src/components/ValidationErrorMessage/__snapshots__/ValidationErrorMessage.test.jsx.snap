// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ValidationErrorMessage renders correctly 1`] = `
<span
  className="validationErrorMessage"
>
  Validation error message
</span>
`;
